.circleci/config.yml

```diff
@@ -206,9 +206,18 @@ jobs:
               poldracklab/fmriprep:latest . \
               --doctest-modules --ignore=docs --ignore=setup.py
       - run:
-          name: Test fmriprep-wrapper
+          name: Test fmriprep-wrapper (Python 2)
           command: |
-            pip install future numpy
+            conda create -n py2 python=2.7
+            source activate py2
```

Comment:
Apparently we don't have `conda` inside the Circle containers. @oesteban is there a good way to install a Py2- or Py3-specific environment in these containers besides installing miniconda? (I'm not sure how flexible normal `venv`s are.)